src/main/java/org/cactoos/list/LimitedIterator.java

```diff
@@ -0,0 +1,49 @@
+package org.cactoos.list;
+
+import java.util.Iterator;
+
+/**
+ * Limited iterator.
+ *
```

Comment:
Note that this gives no thread safety guarantees.

(Concurrent calls to next(), will lead to overshooting the limit)